app/controllers/collection_participants_controller.rb

```diff
@@ -14,37 +14,36 @@ def owners_required
     redirect_to collection_participants_path(@collection)
     false
   end
-  
+
   def no_participant
     flash[:error] = t('no_participant', :default => "Which participant did you want to work with?")
     redirect_to root_path
   end
-  
+
   def load_participant_and_collection
     if params[:collection_participant]
-      @participant = CollectionParticipant.find(params[:collection_participant][:id])
```

Comment:
Can we use find_by_id rather than where ?

http://stackoverflow.com/questions/11161663/find-vs-find-by-vs-where

( soon to be find_by(id: )